tests/assets/testpackage/.brokencontractlayers.ini

```diff
@@ -0,0 +1,15 @@
+[importlinter]
+root_package = testpackage
+format = filenames
```

Comment:
I think it would be better to provide the `format` as a CLI argument when running `lint-imports`, in much the same way as we pass `--show-timings` or `--verbose`. Do you agree?

https://import-linter.readthedocs.io/en/stable/usage.html#running-the-linter